cross-chain/arbitrum/external/goerliEthereum/TokenBridge.json

```diff
@@ -0,0 +1,74 @@
+{
+  "address": "0x23908A62110e21C04F3A4e011d24F901F911744A",
```

Comment:
Double-checked with https://book.wormhole.com/reference/contracts.html#token-bridge-1 but looks the address here is wrong and it should be `0xF890982f9310df57d00f659cf4fd87e65adEd8d7`. Is that correct?

Reply:
Good catch [e4ffa9b](https://github.com/keep-network/tbtc-v2/pull/554/commits/e4ffa9b6c69c782a4694a499ca6300c44428b328)